openedx/features/edly/middleware.py

```diff
@@ -27,6 +31,28 @@ def process_request(self, request):
         if request.user.is_superuser or request.user.is_staff:
             return
 
+        edly_sub_org = getattr(request.site, 'edly_sub_org_for_studio', None)
+        edly_sub_org = getattr(request.site, 'edly_sub_org_for_lms', edly_sub_org)
```

Comment:
We only need the `edly_sub_org_for_lms`. Reason: When the user accesses the studio, Its redirected to LMS for the login same for the preview. In simple if we simple add the check for the LMS, users will not able to loggedIn and eventually it will not able to access our platform.  

cc: @taimoor-ahmed-1

Reply:
Done.